epoxy-processortest/src/test/java/com/airbnb/epoxy/GuavaPatch.kt

```diff
@@ -0,0 +1,22 @@
+package com.airbnb.epoxy
+
+import java.io.File
+import java.net.URL
+
+/**
+ * Since AGP 3.6.0, the class-loader behavior has been modified.
+ * Unfortunately Guava (via compile-testing) uses a class-loader based mechanism
+ * which is valid on JVM but not supposed to be supported on Android.
+ * As the project paths are simple enough, we can hardcode them for now.
+ */
+
+// For Kotlin
+fun String.patchResource(): URL =
+    File("build/intermediates/sourceFolderJavaResources/debug/$this").toURI().toURL()
+
+// For Java
+object GuavaPatch {
```

Comment:
you can add `@file:JvmName("GuavaPatch")` to the top of the file instead of making this object for Java